sepal_ui/bin/activate_venv

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python3
+import subprocess
+from pathlib import Path
+import pandas as pd
+from colorama import init, Fore
+
+# init colors for all plateforms
+init()
+
+if __name__ == "__main__":
+
+    print(
+        f"{Fore.CYAN} Welcome to the virtual env activation, loading your venvs... \n{Fore.RESET}"
+    )
+    
+    # Get usr venvs
+    result = subprocess.run(["jupyter", "kernelspec", "list"], stdout=subprocess.PIPE)
+
+    venvs = pd.DataFrame(
+        [
+            line.split()
+            for line in result.stdout.decode("utf-8").splitlines()
+            if ".local" not in line
+        ],
+    ).iloc[1:]
+
+    # Check if user has test envs
+    test_venv_path = Path.home() / "module-venv"
+    if test_venv_path.exists():
+        test_envs = pd.DataFrame(
+            list(
+                [f"test {el.stem}", str(el)]
```

Comment:
I think you should use `el.name` to catch venv name that include a "dot" like "se.plan"